src/lib/kdf/sp800_56a/sp800_56a.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+* KDF defined in NIST SP 800-56a (Approved Alternative 1)
+*
+* (C) 2017 Ribose Inc.
+* Written by Krzysztof Kwiatkowski.
```

Comment:
Could you write that into one line to make it easier to add years and additional authors.